--- origsrc/gnome-mag-0.15.0/colorblind/keybinder/Makefile.am	2007-12-16 14:11:21.000000000 -0600
+++ src/gnome-mag-0.15.0/colorblind/keybinder/Makefile.am	2008-06-03 00:55:42.296875000 -0500
@@ -2,8 +2,8 @@
 
 _keybinderdir = $(pythondir)/colorblind/keybinder
 _keybinder_LTLIBRARIES = _keybinder.la
-_keybinder_la_LDFLAGS = -module -avoid-version -export-symbols-regex init_keybinder
-_keybinder_la_LIBADD = $(CBAPPLET_LIBS)
+_keybinder_la_LDFLAGS = -module -avoid-version -no-undefined -export-symbols-regex init_keybinder
+_keybinder_la_LIBADD = $(CBAPPLET_LIBS) $(PYTHON_LIBS)
 _keybinder_la_SOURCES = \
 	_keybindermodule.c \
 	eggaccelerators.c \
--- origsrc/gnome-mag-0.15.0/colorblind/osutils/Makefile.am	2007-12-16 14:11:17.000000000 -0600
+++ src/gnome-mag-0.15.0/colorblind/osutils/Makefile.am	2008-06-03 00:55:42.312500000 -0500
@@ -2,8 +2,8 @@
 
 _osutilsdir = $(pythondir)/colorblind/osutils
 _osutils_LTLIBRARIES = _osutils.la
-_osutils_la_LDFLAGS = -module -avoid-version init_osutils
-_osutils_la_LIBADD = $(CBAPPLET_LIBS)
+_osutils_la_LDFLAGS = -module -avoid-version -no-undefined init_osutils
+_osutils_la_LIBADD = $(CBAPPLET_LIBS) $(PYTHON_LIBS)
 _osutils_la_SOURCES = \
 	_osutilsmodule.c
 
--- origsrc/gnome-mag-0.15.0/m4/python.m4	2007-12-16 14:11:17.000000000 -0600
+++ src/gnome-mag-0.15.0/m4/python.m4	2008-06-03 00:55:42.500000000 -0500
@@ -49,7 +49,9 @@
 if test "$py_prefix" != "$py_exec_prefix"; then
   PYTHON_INCLUDES="$PYTHON_INCLUDES -I${py_exec_prefix}/include/python${PYTHON_VERSION}"
 fi
+PYTHON_LIBS="-L${py_exec_prefix}/lib/python${PYTHON_VERSION}/config -lpython${PYTHON_VERSION}"
 AC_SUBST(PYTHON_INCLUDES)
+AC_SUBST(PYTHON_LIBS)
 dnl check if the headers exist:
 save_CPPFLAGS="$CPPFLAGS"
 CPPFLAGS="$CPPFLAGS $PYTHON_INCLUDES"
